browser-test/src/security.test.ts

```diff
@@ -7,7 +7,9 @@ test.describe(
   () => {
     test('applicant cannot access admin pages', async ({page}) => {
       const response = await page.goto('/admin/programs')
-      expect(response!.status()).toBe(403)
+      expect(response!.status()).toBe(200)
+      // Redirected to a non-admin page
+      expect(response!.url()).not.toContain('/admin')
```

Comment:
(non-blocking) If you want to speed this test up, replace `async ({page})` with `async ({request})` and swap over to using [get](https://playwright.dev/docs/api/class-apirequestcontext#api-request-context-get) with [specific response code specific assertions](https://playwright.dev/docs/api/class-apiresponseassertions#api-response-assertions-to-be-ok). 

It will be faster since it will skips clearing the database.

Reply:
Thanks, done.